src/parser/SparqlParser.cpp

```diff
@@ -725,3 +733,76 @@ string SparqlParser::stripAndLowercaseKeywordLiteral(const string& lit) {
   }
   return lit;
 }
+
+// _____________________________________________________________________________
+string SparqlParser::parseLiteral(const string& literal, bool isEntireString,
+                                  size_t off) {
```

Comment:
can you add `size_t off /*defaults to 0*/)` so this is clear without looking into the header?